package.json

```diff
@@ -6,6 +6,15 @@
   "directories": {
     "example": "examples"
   },
+  "private": "true",
+  "workspaces": {
+    "packages": [
```

Comment:
Allow yarn to handle installing for us, minimize time spent and hoist duplicates.

Also allows recompiling locally for shared dependencies.

If this extends to using `tsc --build` then use the [#project-references](https://www.typescriptlang.org/docs/handbook/project-references.html) to build dependencies.